savoirfairelinux/ringme.js
# Add parameters for specifying button defaults and images.

(May not fix, but at least partially addresses #3.)

### Changes proposed in this pull request:

* Add several parameters to `RingMe.ui()` for customizing button creation:
  * `buttonImage`: specify an image URL to use as a button instead of plaintext.
  * `buttonClass`: manually override the classes used on the `<a>` link.
  * `buttonLabel`: manually override the default button text.
* Add a default button image.

### Status


### How to verify this change

* Open the test page.
* Check out the default image being used as a button.
* Edit the call to `RingMe.ui()` as done in the `<script>` tag of `tests/ringme.html` to use some of the new options specified above.

### Additional notes

none
